frontend/modules/shared/ui/src/lib/layout/language-switcher/language-switcher.html

```diff
@@ -0,0 +1,20 @@
+<tui-select
+  [formControl]="language"
+  [tuiTextfieldLabelOutside]="true"
```

Comment:
You can remove `[tuiTextfieldLabelOutside]` when not using label at all

Reply:
when removed the component still leaves space for the label despite not having any which imbalances the flag
also this is copied from the docs